lib/av_client.ts

```diff
@@ -564,9 +553,9 @@ export class AVClient implements IAVClient {
       throw new InvalidStateError('Cannot challenge ballot, no user session')
     }
 
-   let attempts = 0;
-   
-   const executePoll = async (resolve, reject) => {
```

Comment:
This should have been indented in the other direction

Reply:
You're right. I'll fix it right away